Doc/using/cmdline.rst

```diff
@@ -59,6 +59,7 @@ all consecutive arguments will end up in :data:`sys.argv` -- note that the first
 element, subscript zero (``sys.argv[0]``), is a string reflecting the program's
 source.
 
+.. _using-on-interface-option-c:
```

Comment:
That's not needed, use:

```
:option:`-c <command> <-c>`
```

(I'm not sure if it works for -c because of its ``<command>`` argument, I never tried.)

Reply:
Thanks, I will try that.

Reply:
Works nicely, thank you for that!